Vagrantfile

```diff
@@ -3,17 +3,25 @@
 
 Vagrant.require_version ">= 1.8"
 
+MOUNT_OPTIONS = if Vagrant::Util::Platform.linux? then
```

Comment:
I ended up replacing NFS with `rsync` to get around issues with classes not found during the backend compilation phases. Still unsure of what exactly is going on, but the following diff got me to a working state:

```diff
diff --git a/Vagrantfile b/Vagrantfile
index fcd404d..6723805 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -3,17 +3,14 @@
 
 Vagrant.require_version ">= 1.8"
 
-MOUNT_OPTIONS = if Vagrant::Util::Platform.linux? then
-                  ['rw', 'vers=4', 'tcp', 'nolock']
-                else
-                  ['vers=3', 'udp']
-                end
 Vagrant.configure(2) do |config|
   config.vm.box = "ubuntu/trusty64"
 
   config.vm.synced_folder "~/.aws", "/home/vagrant/.aws"
-  config.vm.synced_folder "~/.ivy2", "/home/vagrant/.ivy2"
-  config.vm.synced_folder "./", "/vagrant", type: "nfs", mount_options: MOUNT_OPTIONS
+  config.vm.synced_folder ".", "/vagrant", type: "rsync",
+    rsync__exclude: [".git/", "app-backend/**/target/",
+                     "deployment/ansible/roles/azavea*/"],
+    rsync__args: ["--verbose", "--archive", "-z"]
 
   config.vm.provider :virtualbox do |vb|
     vb.memory = 8192
diff --git a/docker-compose.yml b/docker-compose.yml
index 2c522e7..d6bbc3e 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -23,8 +23,6 @@ services:
     volumes:
         - ./src/app-frontend:/usr/src
         - ./src/nginx/srv/dist:/usr/src/dist
-        # Ensure node_modules cache doesn't clash with other jobs on CI.
-        - ${PWD}/potsdam-demo-node-modules:/usr/src/node_modules
     entrypoint: npm
     command: run dev-server
   api-server:
@@ -35,7 +33,6 @@ services:
         - ./src/app-backend/:/opt/app-backend/
         - ./data/catalog-hadoop:/opt/catalog-hadoop/
         - ./data/catalog:/opt/catalog/
-        - ./src/app-backend/.sbt:/root/.sbt
         - $HOME/.ivy2/:/root/.ivy2
         - $HOME/.aws:/root/.aws:ro
     environment:
```